metron-deployment/vagrant/full-dev-platform/README.md

```diff
@@ -13,7 +13,9 @@ Getting Started
 The computer used to deploy Apache Metron will need to have the following components installed.
 
  - [Ansible](https://github.com/ansible/ansible) (2.0.0.2 or 2.2.2.0)
+ - [Docker](https://www.docker.com/community-edition)
  - [Vagrant](https://www.vagrantup.com) 1.8.1
+ - [Vagrant Hostmanager Plugin](https://github.com/devopsgroup-io/vagrant-hostmanager) `vagrant plugin install vagrant-hostmanager`
```

Comment:
I don't have a strong opinion on this, but I think the impact is minimal. As Nick stated, it's kind of nice to be able to have the deployment automatically setup a host name that we can use to interact with the env. I believe in our usage it should only append to /etc/hosts. And when you do `vagrant destroy` it will update the file by removing only the line wrapped/tagged with the corresponding ID, e.g.
```
## vagrant-hostmanager-start id: eded1905-6263-4699-8d30-45e30f693db6
192.168.66.121  node1

## vagrant-hostmanager-end
```